pytype/tools/analyze_project/main.py

```diff
@@ -75,12 +75,15 @@ def main():
     import_graph = importlab.graph.ImportGraph.create(
         env, conf.inputs, trim=True)
     unreadable_inputs = conf.inputs & import_graph.unreadable_files
-    assert not unreadable_inputs, '\n  '.join(
+    unreadable_python_inputs = {file for file in unreadable_inputs if file.endswith('.py')}
```

Comment:
Please rename `file` to something else, since pytype is Python 2-compatible, and `file` is a keyword in Python 2.

Reply:
My bad. sorry